src/sign/index.ts

```diff
@@ -0,0 +1,42 @@
+import { getAddress } from '@ethersproject/address';
+import { Wallet, verifyTypedData } from '@ethersproject/wallet';
+import { Subscriber, SubscribeTypes } from './types';
+
+const NAME = 'envelop';
+const VERSION = '1';
+
+const domain = {
+  name: NAME,
+  version: VERSION
+};
+
+const buildValue = (email: string, address: string): Subscriber => {
+  return {
+    email,
+    address: getAddress(address)
+  };
+};
+
+const signer = new Wallet(process.env.WALLET_PRIVATE_KEY as string);
+
+const sign = async (message, type): Promise<string> => {
+  return await signer._signTypedData(domain, type, message);
+};
+
+const verify = (message, signature: string, type): boolean => {
```

Comment:
All the functions should use this format
```suggestion
function verify(message, signature: string, type): boolean {
```